antarest/study/storage/abstract_storage_service.py

```diff
@@ -82,53 +81,29 @@ def patch_update_study_metadata(
     def get_study_information(
         self,
         study: T,
-        summary: bool = False,
     ) -> StudyMetadataDTO:
-        file_settings = {}
-        file_metadata = {}
+        if not study.additional_data:
+            # TODO: remove this after all additional_data has been initialized
+            self.initialize_additional_data(study)
 
-        patch_metadata = PatchStudy()
+        try:
+            patch = Patch.parse_raw(study.additional_data.patch)
```

Comment:
study.additional_data.patch or "{}" to prevent the validation error (lots of studies won't have patches for now)

Reply:
That is actually the purpose of the catch with the ValidationError

Reply:
yeah but the catch has lower performance than an if/else when it goes to the catch block which will definitely do most of the time here